distribution/docs/src/main/resources/_contents/configuring-from-the-command-console-contents.adoc

```diff
@@ -0,0 +1,164 @@
+====
+
+Templates included with ${ddf-branding}:
+
+[cols="1,4,4,1" options="header"]
+|===
+
+|${ddf-branding} Service
+|Template File Name
+|Factory PID
+|Configurable Properties
+
+|${ddf-branding} Catalog Framework
+|`${ddf-branding-lowercase}.catalog.impl.service.CatalogFrameworkImpl.cfg`
+|`${ddf-branding-lowercase}.catalog.CatalogFrameworkImpl`
+|Standard Catalog Framework
+
+|===
+
+===== Configuring Using a `.cfg` File Template
+
+The following steps define the procedure for configuring a new source or feature using a `config` file.
+
+. Copy/rename the provided template file in the ``etc/templates` directory to the `etc` directory. (Refer to the table above to determine correct template.)
+.. *Mandatory*: The dash between the PID (e.g., `OpenSearchSource_site.cfg`) and the instance name (e.g., `OpenSearchSource_site.cfg`) is required. The dash is a reserved character used by OSGi that identifies instances of a managed service factory that should be created.
+.. Not required, but a good practice is to change the instance name (e.g., `federated_source`) of the file to something identifiable (`source1- ddf`).
+. Edit the copied file to etc with the settings for the configuration. (Refer to the table above to determine the configurable properties).
+.. This file is a Java properties file, hence the syntax is `<key>` = `<value>`.
+.. Consult the inline comments in the file for guidance on what to modify.
+.. The Configurable Properties tables in the Integrator's Guide for the Included Catalog Components also describe each field and its value.
+
+The new service can now be used as if it was created using the {admin-console}.
+
+===== Managing Applications From the {command-console}
+
+Applications can be installed from the {command-console} using the following commands:
+
+.App Commands
+[cols="2,5" options="header"]
+|===
+|Command
+|Effect
+
+|`app:add <appName>`
+|Install an app.
+
+|`app:list`
+|List all installed apps and current status.
+
+|`app:remove <appName>`
+|Uninstall an app.
+
+|`app:start`
+|Start an inactive app.
+
+|`app:status <appName>`
+|Detailed view of application status
+
+|`app:stop <appName>`
+|Stop an active app.
+
+|`app:tree`
+|Dependency tree view of all installed apps.
+|===
+
+===== Managing Features From the {command-console}
+
+. Determine which feature to install by viewing the available features on ${branding}. +
+`${branding-lowercase}${at-symbol}local>features:list`
+. The console outputs a list of all features available (installed and uninstalled). A snippet of the list output is shown below (the versions may differ):
+
+----
+State         Version          Name                          Repository  Description
+[installed  ] [2.0.1         ] ${ddf-branding-lowercase}-core                      ${ddf-branding-lowercase}-2.1.0
```

Comment:
1- The version column (2.0.1 in this case), usually matches the repository version... so I would probably choose between 2.0.1 and 2.1.0 and be consistent ;)
2- The "Repository" header does not line up correctly here (it does if you run it on a ddf)
3- The "Description" header does not line up either and the column is empty otherwise... should we add sample descriptions? Or maybe pull from a recent DDF like this (but remove the -SNAPSHOT):
```
State         Version            Name                                     Repository                           Description
[installed  ] [2.9.0-SNAPSHOT  ] security-handler-api                     security-services-app-2.9.0-SNAPSHOT API for authentication handlers for web applications.
[installed  ] [2.9.0-SNAPSHOT  ] security-core                            security-services-app-2.9.0-SNAPSHOT DDF Security Core
[uninstalled] [2.9.0-SNAPSHOT  ] security-expansion                       security-services-app-2.9.0-SNAPSHOT DDF Security Expansion
[uninstalled] [2.9.0-SNAPSHOT  ] security-cas-client                      security-services-app-2.9.0-SNAPSHOT DDF Security CAS Client.
[uninstalled] [2.9.0-SNAPSHOT  ] security-cas-tokenvalidator              security-services-app-2.9.0-SNAPSHOT DDF Security CAS Validator for the STS.
[uninstalled] [2.9.0-SNAPSHOT  ] security-cas-cxfservletfilter            security-services-app-2.9.0-SNAPSHOT DDF Security CAS Servlet Filter for CXF.
[installed  ] [2.9.0-SNAPSHOT  ] security-pdp-authz                       security-services-app-2.9.0-SNAPSHOT DDF Security PDP.
[uninstalled] [2.9.0-SNAPSHOT  ] security-pep-serviceauthz                security-services-app-2.9.0-SNAPSHOT DDF Security PEP Service AuthZ
[uninstalled] [2.9.0-SNAPSHOT  ] security-pep-user-attributes             security-services-app-2.9.0-SNAPSHOT DDF Security PEP User Attributes Expansion
[uninstalled] [2.9.0-SNAPSHOT  ] security-pep-metacard-attributes         security-services-app-2.9.0-SNAPSHOT DDF Security PEP Metacard Attributes Expansion
[installed  ] [2.9.0-SNAPSHOT  ] security-sts-server                      security-services-app-2.9.0-SNAPSHOT DDF Security STS.
[installed  ] [2.9.0-SNAPSHOT  ] security-sts-realm                       security-services-app-2.9.0-SNAPSHOT DDF Security STS Realm.
[uninstalled] [2.9.0-SNAPSHOT  ] security-sts-ldaplogin                   security-services-app-2.9.0-SNAPSHOT DDF Security STS JAAS LDAP Login.
[uninstalled] [2.9.0-SNAPSHOT  ] security-sts-ldapclaimshandler           security-services-app-2.9.0-SNAPSHOT Retrieves claims attributes from an LDAP store.
```